### PR TITLE
Unit Tests use a consistent human subtype

### PR DESCRIPTION
**Pull request by ZephyrTFA**
title

### CODE REVIEW

**Comment by Mothblocks** [comment]
the [code/modules/mob/living/carbon/human/consistent.dm](https://github.com/tgstation/tgstation/pull/71515/files#diff-160c5ee97be0ae889b213f53a2e045fde6fa01f215b420c64f20f636e79f346a) file is empty

File: code/modules/mob/living/carbon/human/dummy.dm [hunk 1]
```diff
@@ -103,6 +103,33 @@ INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy)
 	dna.features["tail_lizard"] = "Smooth"
 	dna.features["pod_hair"] = "Ivy"
 
+/// Provides a dummy for unit_tests that functions like a normal human, but with a standardized appearance
+/// Copies the stock dna setup from the dummy/consistent type
+/mob/living/carbon/human/unit_test
```
Comment (Mothblocks) [inline code/modules/mob/living/carbon/human/dummy.dm:108]: Name it `consistent` instead to be...consistent.

File: code/modules/mob/living/carbon/human/dummy.dm [hunk 2]
```diff
@@ -103,6 +103,33 @@ INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy)
...
+
+/mob/living/carbon/human/unit_test/setup_human_dna()
+	create_dna(src)
+	dna.initialize_dna(skip_index = TRUE)
```
Comment (Mothblocks) [inline code/modules/mob/living/carbon/human/dummy.dm:112]: This needs to be a proc instead of a copy paste, I change these often enough to warrant it

**Comment by ZephyrTFA** [comment]
Yeah I was gonna put it in its own file originally then I figured it wasn't really worth it

**Review by Mothblocks — CHANGES_REQUESTED** [review]

File: code/modules/unit_tests/mob_spawn.dm [hunk 3]
```diff
@@ -19,7 +19,8 @@
 		if(ghost_role.outfit_override)
 			TEST_FAIL("[ghost_role.type] has a defined \"outfit_override\" list, which is only for mapping. Do not set this!")
 
-		if(ghost_role.mob_type != /mob/living/carbon/human/consistent)
+		var/human_type = /mob/living/carbon/human // this exists to prevent being caught by checks
+		if(ghost_role.mob_type != human_type)
```
Comment (Mothblocks) [inline code/modules/unit_tests/mob_spawn.dm:23]: Feels like we should just be doing `ishuman` instead

**Review by Mothblocks — APPROVED** [review]

**Pull request merged**